app/assets/javascripts/map.js.coffee

```diff
@@ -105,6 +122,25 @@ App.Map =
 
           marker.on "click", openMarkerPopup
 
+    if addMarkerProposals
+      relatedMarkers = L.layerGroup()
+      for i in addMarkerProposals
```

Comment:
I know we're copying the code from the previous lines. However, those lines were changed in consul@1cc3324468 :thinking:. How about applying that commit first, and then using `forEach` in the new code?